tests/e2e/config/playwright.config.js

```diff
@@ -0,0 +1,69 @@
+const { defineConfig, devices } = require( '@playwright/test' );
+const { url } = require( './default.json' );
+
+module.exports = defineConfig( {
+	testDir: '../specs',
+
+	/* Maximum time one test can run for. */
+	timeout: 120 * 1000,
+
+	expect: {
+		/**
+		 * Maximum time expect() should wait for the condition to be met.
+		 * For example in `await expect(locator).toHaveText();`
+		 */
+		timeout: 20 * 1000,
+	},
+
+	/* Fail the build on CI if you accidentally left test.only in the source code. */
+	forbidOnly: !! process.env.CI,
+
+	/* Retry on CI only */
+	retries: process.env.CI ? 2 : 0,
+
+	/* Reporter to use. See https://playwright.dev/docs/test-reporters */
+	reporter: [
+		[ 'list' ],
+		[
+			'html',
+			{
+				outputFolder: '../test-results/playwright-report',
+				open: 'never',
+			},
+		],
+	],
+
+	/* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+	use: {
+		/* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
```

Comment:
/* Maximum time in milliseconds each action such as `click()` can take. Defaults to 0 (no limit). */